interfaces/cython/cantera/_onedim.pyx

```diff
@@ -733,6 +733,20 @@ cdef class FreeFlow(_FlowBase):
     _domain_type = "free-flow"
 
 
+cdef class StabilizedFlow(_FlowBase):
+    r"""A stabilized flow domain. The equations solved are standard equations for
+    adiabatic one-dimensional flow. The solution variables are:
```

Comment:
All of these domains provide a way of "stabilizing" the flame. What about calling this one "UnstrainedFlow", which would at least make it clear that what's being excluded is the possibility of a strained flame?

Reply:
Nomenclature is indeed tricky. I ended up with `stabilized` as it's part of the burner-stabilized terminology, although `parallel-flow` was my initial hunch. `free-flow` is both parallel and unstrained. I also considered `fixed-flow` ... what's even more aggravating is that all could be considered `axisymmetric`. I'm :+1: with unstrained, as it does get buried in `BurnerFlame` anyways.

Reply:
Yeah, I was trying not to bring up `axisymmetric` and open up any more cans of worms. 😄